src/Engine/Basis.h

```diff
@@ -635,10 +635,23 @@ class Basis
 				UnonConst(row, col) = Uconst(row, perm[col]);
 	}
 
-	friend std::istream& operator>>(std::istream& is,
-	    Basis<SparseMatrixType>&)
-	{
-		throw PsimagLite::RuntimeError("Unimplemented >>");
+	//! The operator<< is a friend
+	friend std::ostream& operator<<(std::ostream& os,
+	    const Basis<SparseMatrixType>& x)
```

Comment:
Did you mean to commit that change?
Why is it ok to drop the operator>> ?

Reply:
I'm not dropping it I had fixed it so it still worked the features branch had not, I'm guessing just because @g1257 was saving time.  If he removed the output for a reason other than it had stopped compiling he should indicate it should go back to throwing and exception.  I think if its being dropped it would be better just to remove it and have compiles that are trying to use it fail.

Reply:
OK